tools/Google.Cloud.Docs.Snippets/Google.Cloud.Docs.Snippets.csproj

```diff
@@ -22,10 +22,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Apache.Avro" Version="1.11.3" />      
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(TestSdkVersion)" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="$(XUnitRunnerVersion)" />
-    <PackageReference Include="xunit" Version="$(XUnitVersion)" />
+    <PackageReference Include="Apache.Avro" VersionOverride="1.11.3" />      
```

Comment:
Apache.Avro is not on the list of dependencies. Do we still need to use VersionOverride?